src/containers/ProfileDetails/constants/index.js

```diff
@@ -2,6 +2,13 @@ export const RADIOS = ['is2dot4GHz', 'is5GHz', 'is5GHzU', 'is5GHzL'];
 export const ROAMING = ['enable80211r', 'enable80211k', 'enable80211v'];
 export const DEFAULT_NTP_SERVER = 'pool.ntp.org';
 export const DEFAULT_HESS_ID = '00:00:00:00:00:00';
+export const EAP_METHODS = {
```

Comment:
instead you can just change the values of the Options to this

Reply:
I verified with the yaml file and we don't need to do this conversion, so I removed this object